app.js

```diff
@@ -1,11 +1,79 @@
-/* Imports */
-
 /* Get DOM Elements */
+const farmSelect = document.getElementById('farm-select');
+const farmImg = document.getElementById('farm-img');
+const farmChangeCounter = document.getElementById('farm-change-counter');
+
+const professionSelect = document.getElementById('profession-select');
+const professionImg = document.getElementById('profession-img');
+const professionChangeCounter = document.getElementById('profession-change-counter');
+
+const romanceSelect = document.getElementById('romance-select');
+const romanceImg = document.getElementById('romance-img');
+const romanceChangeCounter = document.getElementById('romance-change-counter');
+
+const favoriteInput = document.getElementById('favorite-input');
+const okButton = document.getElementById('ok-button');
+const favoriteList = document.getElementById('favorite-list');
 
 /* State */
+let farmCounter = 0;
+let professionCounter = 0;
+let romanceCounter = 0;
+let favoriteArray = [];
 
 /* Events */
+farmSelect.addEventListener('change', () => {
+    const id = farmSelect.value;
+
+    farmImg.src = `./assets/farm-${id}.png`;
+    farmImg.alt = `${id} farm thumbnail`;
+
+    farmCounter++;
+
+    farmChangeCounter.textContent = `You've changed farms ${farmCounter} times`;
```

Comment:
This a great way to change the state but it is a little repetitive.  Any way to condense this?